### DISCUSSION TITLE
Github subscription issue - copilot

### DISCUSSION THREAD

**Discussion by AyodejiO** [community/community#56550] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Why would I get this error in vscode after paying the annual fee for GH Copilot? 
```No access to GitHub Copilot found. You are currently logged in as AyodejiO.```

**Comment by davecheney** [comment]
@AyodejiO there is no Copilot plan associated with this github username. Did you perhaps use a different account when signing up?

**Reply by AyodejiO** [reply]
No I didn't. I used this one. Should I upload the receipt here?

**Reply by AyodejiO** [reply]
here is the receipt
[github-AyodejiO-receipt-2023-04-22.pdf](https://github.com/community/community/files/11585322/github-AyodejiO-receipt-2023-04-22.pdf)

**Reply by AyodejiO** [reply]
@davecheney

**Reply by davecheney** [reply]
You card was declined. Please contact support https://support.github.com/contact

**Reply by AyodejiO** [reply]
The card was specifically for Copilot, and GH tried to charge for the premium plan. Can't I have one without the other?

**Reply by davecheney** [reply]
Yes, there is no requirement to purchase a Pro plan to use Copilot.

**Reply by AyodejiO** [reply]
Yes, and if that's the case, the card has been debited successfully. Are you suggesting otherwise?

**Reply by davecheney** [reply, marked as answer]
I'm sorry I cannot assist you via this forum, please contact support https://support.github.com/contact

**Reply by AyodejiO** [reply]
Thanks.
